### PR TITLE
Add support for Powerpoint, Word doc, PDF, Outlook email, ZIP, GZ, BZ2

### PR DESCRIPTION
**Pull request by tooptoop4**
Fixes https://github.com/ebyhr/presto-flex/issues/16

### CODE REVIEW

File: src/main/java/org/ebyhr/presto/flex/FlexRecordCursor.java [hunk 1]
```diff
@@ -62,25 +83,234 @@ public FlexRecordCursor(List<FlexColumnHandle> columnHandles, SchemaTableName sc
...
+            try (CountingInputStream input = new CountingInputStream(byteSource.openStream())) {
+                lines = plugin.getIterator(byteSource);
+                if (plugin.skipFirstLine()) {
+                    lines.next();
+                }
+                totalBytes = input.getCount();
+            }
+            catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
```
Comment (ebyhr) [inline src/main/java/org/ebyhr/presto/flex/FlexRecordCursor.java:313]: Please refer to the existing implementation.

File: src/main/java/org/ebyhr/presto/flex/FlexRecordCursor.java [hunk 2]
```diff
@@ -62,25 +83,234 @@ public FlexRecordCursor(List<FlexColumnHandle> columnHandles, SchemaTableName sc
...
+                    lst.add(inputLine);
+                }
+                lines = lst.iterator();
+                totalBytes = input.getCount();
+            }  catch (IOException e) {
+                throw new UncheckedIOException(e);
+            } finally {
+                try { if (in != null)
+                    in.close();
+                } catch (Exception e) { /* ignored */ }
```
Comment (ebyhr) [inline src/main/java/org/ebyhr/presto/flex/FlexRecordCursor.java:279]: Fix broken indentation.

**Comment by ebyhr** [comment]
Please remove the redundant first commit and resolve conflicts.

**Comment by tooptoop4** [comment]
found some bugs in the readers

**Pull request closed without merging**